lib/linguist/languages.yml

```diff
@@ -5580,6 +5580,15 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+vvvv:
```

Comment:
[It looks like](https://vvvv.org/documentation/vl) vvvv is the name of the software and VL the name of the language, no?

Reply:
No, vvvv is a language unto itself. vl is a relatively new, additional language that can be used within vvvv to solve certain problems more easily. I'd view vvvv and vl as a unit, but if you find that cleaner, I can remove the `.vl` extension from `languages.yml`, treat vl as a completely separate language and use this pull request to only add vvvv and its .v4p files to Linguist.

Reply:
Actually, thinking about it - I think treating them as separate languages would be the best way to go, because they also look a bit different, which is what caused the confusion about the sample file, which is pure vvvv. I'll remove the .vl extension in the next commit.